content/en/blog/news/five-new-things/index.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2024-11-08
+title: Five New Things in SpinKube
+linkTitle: Five New Things in SpinKube
+description: >
+  Catching up on what's new in SpinKube
+author: The SpinKube Team ([@SpinKube](https://mastodon.social/@SpinKube))
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+
+Since we publicly [released](/blog/2024/03/13/introducing-spinkube/) SpinKube in March we've been hard at work steadily making it better. Spin Operator [`v0.4.0`](https://github.com/spinkube/spin-operator/releases/tag/v0.4.0), ContainerD shim for Spin [`v0.17.0`](TODO), and `spin kube` plugin [`v0.3.0`](https://github.com/spinkube/spin-plugin-kube/releases/tag/v0.3.0) have all just been released. To celebrate that, here's five new things in SpinKube you should know about.
+
+## Selective Deployments
+
+SpinKube now supports selectively deploying a subset of a Spin apps components. Here's an example of what the YAML for a selectively deployed app might look like:
```

Comment:
May want to add a bit of context on the sample app to make sure the selective deployment messaging hits home. 

```suggestion
SpinKube now supports selectively deploying a subset of a Spin apps components. Consider this simple example Spin application (named salutation in the [example repo](https://github.com/spinkube/spin-operator/tree/main/apps/salutations)) composed of two HTTP-triggered components: `hello` and `goodbye`. Here's an example of what the YAML for a selectively deployed app might look like:
```